Automattic/jetpack
# AI Inline Extension: Move Jetpack Form

Closes #37590 

## Proposed changes:

* Add support for Jetpack Form and Children on `inline-extensions`.
* Add `suggestion` parameter at `onDone` callback.
* Add support for custom `feature` on request suggestion.
* Add support to disable `adjustPosition`.
* Add support to start open.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Open your editor and make sure you enable support for unreleased inline extensions.
* Add form block, you should see the new input and interact with it.
* Clicking on children will open the input at the form block.

### Demo

https://github.com/Automattic/jetpack/assets/1663717/db802a9e-8eff-4a65-aede-cf970e48035d
